### ISSUE TITLE
Emulated Read Speed causes various problems

### ISSUE THREAD

**Issue by kingjinxy** [emukidid/swiss-gc#414]
Swiss: 0.5r922
Device: GCLoader
GameCube: DOL-001 NTSC
Video: Composite NTSC
ISO type: Redump-verified full ISOs
SD Card: Samsung 128GB microSD

Issue(s):

I have been testing various games with Emulated Read Speed activated and have found issues that do not otherwise occur:

Mario Kart - Double Dash!! Bonus Disc: It boots fine, but selecting a movie (or possibly a game) will send me back to Swiss. 

Super Smash Bros. Melee: Playing the "How to Play" movie results in stuttering and a desync of audio and video. This may also happen with the "Special Movie".

The Legend of Zelda: Collector's Edition: Selecting a game sends me back to Swiss.

The Legend of Zelda: Ocarina of Time & Master Quest: Selecting a game sends me back to Swiss.

**Comment by Extrems** [comment]
From The Legend of Zelda: Ocarina of Time & Master Quest:
```
Dolphin OS $Revision: 47 $.
Kernel built : Nov 20 2001 11:01:20
Console Type : Development HW3
Memory 24 MB
Arena : 0x81324e80 - 0x81700000
app booted from bootrom

--- Game loader ---
OSSetAlarm(): tick was less than zero. in "OSAlarm.c" on line 283.

Address:      Back Chain    LR Save
0x81322958:   0x813229e8    0x81303038
0x813229e8:   0x81322a20    0x81301b7c
0x81322a20:   0x81322d08    0x80001c80
0x81322d08:   0x81322d20    0x80002360
0x81322d20:   0x81322d28    0x80000c6c
0x81322d28:   0x81322d48    0x81307ef8
0x81322d48:   0x81322d60    0x813080b0
0x81322d60:   0x81322da0    0x813082d4
0x81322da0:   0x81322db8    0x8130a564
0x81322db8:   0x81322dc8    0x8130a478
0x81322dc8:   0x81322de8    0x8130ad68
0x81322de8:   0x81322e20    0x8130ae70
0x81322e20:   0x81322e50    0x81300958
0x81322e50:   0x81322e60    0x81300484
0x81322e60:   0xffffffff    0x81300094
```

**Comment by Extrems** [comment]
Fixed in 1a1698c (r961).
